src/Admin/AMPThemes.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/**
+ * Add new tab (AMP) in theme install screen in WordPress admin.
+ *
+ * @package Ampproject\Ampwp
+ */
+
+namespace AmpProject\AmpWP\Admin;
+
+use AmpProject\AmpWP\Infrastructure\Registerable;
+use AmpProject\AmpWP\Infrastructure\Service;
+use WP_Filesystem_Base;
+
+/**
+ * Add new tab (AMP) in theme install screen in WordPress admin.
+ *
+ * @since 2.2
+ * @internal
+ */
+class AMPThemes implements Service, Registerable {
+
+	/**
+	 * Assets handle.
+	 *
+	 * @var string
+	 */
+	const ASSET_HANDLE = 'amp-theme-install';
+
+	/**
+	 * @var array List of AMP themes.
+	 */
+	public static $themes = [];
+
+	/**
+	 * To initialize file system.
+	 *
+	 * @return void
+	 */
+	public static function init_file_system() {
+		global $wp_filesystem;
+
+		require_once ABSPATH . 'wp-admin/includes/file.php';
+
+		if ( ! $wp_filesystem instanceof WP_Filesystem_Base ) {
+			$creds = request_filesystem_credentials( site_url() );
+			wp_filesystem( $creds );
+		}
+	}
+
+	/**
+	 * Fetch AMP themes data.
+	 *
+	 * @return void
+	 */
+	public static function set_themes() {
+		global $wp_filesystem;
+
+		self::init_file_system();
+
+		$file_path = AMP__DIR__ . '/data/themes.json';
+
+		if ( ! file_exists( $file_path ) ) {
+			return;
+		}
+
+		$json_data       = $wp_filesystem->get_contents( $file_path );
+		self::$themes    = json_decode( $json_data, true );
+		$json_last_error = json_last_error();
+
+		if ( JSON_ERROR_NONE !== $json_last_error ) {
+			self::$themes = [];
+		}
+	}
+
+	/**
+	 * Adds hooks.
+	 *
+	 * @return void
+	 */
+	public function register() {
+
+		self::set_themes();
+
+		add_filter( 'themes_api', [ $this, 'themes_api' ], 10, 3 );
+
+		if ( ! wp_doing_ajax() && is_admin() ) {
+			add_action( 'current_screen', [ $this, 'register_hooks' ] );
+		}
+	}
+
+	/**
+	 * Register all hooks.
+	 *
+	 * @return void
+	 */
+	public function register_hooks() {
+
+		$screen = get_current_screen();
+
+		if ( $screen instanceof \WP_Screen && in_array( $screen->id, [ 'themes', 'theme-install' ], true ) ) {
```

Comment:
Add a `use WP_Screen`:

```suggestion
		if ( $screen instanceof WP_Screen && in_array( $screen->id, [ 'themes', 'theme-install' ], true ) ) {
```